spec/controllers/response_controller_spec.rb

```diff
@@ -362,4 +362,11 @@
     end
   end
 
+  describe '#fetch_review_metric' do
+    context 'fetches the review metrics from review_metric.yml file' do
+      it 'returns a list of configuration' do
+        expect(fetch_review_metric).not_to be_nil
```

Comment:
very shallow test. make sure the yml file is read with what you expect it to be

Reply:
metrics = fetch_review_metric #["sentiment","problem","suggestion"], ["sentiment"], []
expect(metrics.length) to be <= 3
metrics.each do |metric|
expect(["sentiment","problem","suggestion"].include? metric).to eq(true)
end